text/0000-clarity-api-description.md

```diff
@@ -47,18 +41,8 @@ This type of transition, though, should have a URL and a method.
 
 We should add a section that defines what media types the server will respond with.
 
-## Add Schema to Resource
-
-A resource should be able to have its own schema, whereas now it cannot.
-
 # Unresolved questions
 
-## Category Element
-
-What is a "category"?
```

Comment:
The description seems pretty clear to me. I believe the question was more around the naming.